notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T231-analysis-individual_hp_breakaway.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<p>The SITCOMTN-084 notebook seems very complete. Is there any reason of why we want to keep this one? If so, can we write that in the description? This is also related with the TN, so maybe change the prefix of the notebook file as well? We can keep everything else the same. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>